src/topology/algebra/uniform_group.lean

```diff
@@ -230,6 +230,62 @@ variables {G}
 ⟨λ h u hu, h _ ⟨u, hu, λ _, id⟩, λ h v ⟨u, hu, hv⟩ x, exists_imp_exists (by exact λ a,
   exists_imp_exists (λ ha hp, mem_of_superset hp (λ i hi a ha, hv (by exact hi a ha)))) ∘ h u hu x⟩
 
+@[to_additive] lemma topological_group.uniform_continuous_mul :
+  let t := topological_group.to_uniform_space G in @uniform_continuous₂ G G G t t t (*) :=
+begin
+  rintros t v ⟨u, hu, huv⟩,
+  obtain ⟨w, hw, x, hx, h⟩ := mem_nhds_prod_iff.mp (mem_map.mp
+    (continuous_mul.tendsto' ((1 : G), (1 : G)) (1 : G) (one_mul (1 : G)) hu)),
+  rw [uniformity_prod_eq_prod, mem_map, mem_map, mem_prod_iff],
+  refine ⟨_, preimage_mem_comap hw, _, preimage_mem_comap hx, _⟩,
+  rw [set.prod_preimage_eq],
+  exact λ a ha, huv (by exact ((congr_arg (∈ u) (div_mul_comm _ _ _ _)).mp (h ha))),
+end
+
+@[to_additive] lemma topological_group.tendsto_uniformly_on_mul
+  {ι₁ ι₂ α : Type*} (F₁ : ι₁ → α → G) (F₂ : ι₂ → α → G)
+  (f₁ : α → G) (f₂ : α → G) (p₁ : filter ι₁) (p₂ : filter ι₂) (s : set α)
+  (h₁ : @tendsto_uniformly_on α G ι₁ (topological_group.to_uniform_space G) F₁ f₁ p₁ s)
+  (h₂ : @tendsto_uniformly_on α G ι₂ (topological_group.to_uniform_space G) F₂ f₂ p₂ s) :
+  @tendsto_uniformly_on α G (ι₁ × ι₂) (topological_group.to_uniform_space G)
+    (λ i, F₁ i.1 * F₂ i.2) (f₁ * f₂) (p₁.prod p₂) s :=
+begin
+  letI := topological_group.to_uniform_space G,
+  exact (h₁.prod h₂).comp' topological_group.uniform_continuous_mul,
+end
+
+@[to_additive] lemma topological_group.tendsto_uniformly_on_inv {ι α : Type*} (F : ι → α → G)
```

Comment:
Can we factor this result through the fact that inversion is uniformly continuous, like we did for multiplication?

Reply:
Done.